release-notes/9.0/preview/preview1/api-diff/Microsoft.NETCore.App/9.0-preview1_System.Runtime.InteropServices.Marshalling.md

```diff
@@ -0,0 +1,28 @@
+# System.Runtime.InteropServices.Marshalling
+
+``` diff
+ namespace System.Runtime.InteropServices.Marshalling {
++    public struct ComVariant : IDisposable {
++        public static ComVariant Null { get; }
++        public VarEnum VarType { get; }
++        public T? As<T>();
++        public static ComVariant Create<T>(T value);
++        public static ComVariant CreateRaw<T>(VarEnum vt, T rawValue) where T : struct;
```

Comment:
```suggestion
+        public static ComVariant CreateRaw<T>(VarEnum vt, T rawValue) where T : unmanaged;
```

Reply:
@jkoritzinsky I opened an issue to track your two suggestions: https://github.com/dotnet/arcade/issues/14497